runtime-watcher/hack/release.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# standard bash error handling
+set -o nounset # treat unset variables as an error and exit immediately.
+set -o errexit # exit immediately when a command fails.
+set -E         # needs to be set if we want the ERR trap
+
+get_new_release_version() {
+    # get the list of tags in a reverse chronological order
+    TAG_LIST=($(git tag --sort=-creatordate))
+    export GORELEASER_CURRENT_TAG=${TAG_LIST[0]}
+}
+
+get_previous_release_version() {
+    # get the list of tags in a reverse chronological order excluding release candidate tags
+    TAG_LIST_WITHOUT_RC=($(git tag --sort=-creatordate | grep -v -e "-rc"))
+    if [[ $GORELEASER_CURRENT_TAG == *"-rc"* ]]; then
+        export GORELEASER_PREVIOUS_TAG=${TAG_LIST_WITHOUT_RC[0]}
+    else
+        if [[ ${#TAG_LIST_WITHOUT_RC[@]} -gt 1 ]]; then
+            export GORELEASER_PREVIOUS_TAG=${TAG_LIST_WITHOUT_RC[1]}
+        else
+            export GORELEASER_PREVIOUS_TAG=""
+        fi
+    fi
+}
+
+main() {
+    git remote add origin git@github.com:kyma-project/runtime-watcher.git
+    get_new_release_version
+    get_previous_release_version
+    # release watcher with release notes generated by goreleaser
+    curl -sL https://git.io/goreleaser | VERSION=v1.17.2 bash -s --
```

Comment:
Lets directly use latest one

```suggestion
    curl -sL https://git.io/goreleaser | VERSION=v1.21.2 bash -s --
```